desktop/src/main/java/bisq/desktop/main/account/content/altcoinaccounts/AltCoinAccountsView.java

```diff
@@ -147,6 +147,11 @@ private void onSaveNewAccount(PaymentAccount paymentAccount) {
                             .useIUnderstandButton()
                             .show();
                     break;
+                case "LTZ":
+                    new Popup<>().information(Res.get("account.altcoin.popup.LTZ.msg", "LTZ"))
```

Comment:
Does this message bring any added value at this point?
It duplicates validation error.

Reply:
If you see messages are quite similar to ZEC but customized for LTZ:

account.altcoin.popup.LTZ.msg=When using {0} you can only use the transparent addresses (`starting with L`) not the z-addresses (private), because the arbitrator would not be able to verify the transaction with z-addresses.

account.altcoin.popup.ZEC.msg=When using {0} you can only use the transparent addresses (`starting with t`) not the z-addresses (private), because the arbitrator would not be able to verify the transaction with z-addresses.

Reply:
I know, but still it doesn't add any value to repeat validation message.
I value your effort with all those translations.
Team will have to discuss if post save popups with this type of messages have any sense.

Reply:
Well the sense is to advice customers to use only transparent address and not z-address because they are not supported. Now, because LTZ use address starting with 'L' and not with 't'  like ZEC does